docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md

```diff
@@ -5,8 +5,7 @@ author: Dustin Schau
 excerpt: "In this post, I outline the tasks, responsibilities, and day-to-day happenings of a Core contributor"
 tags:
   - community
-  - open source
-  - contributor
+  - gatsby-inc
```

Comment:
```suggestion
  - gatsby-inc
  - open-source
```
I'd recommend keeping this, as it's another area that shows how company and OSS overlap: Gatsby employs full-time maintainers to benefit the open source community